content/blog/0035-effect-3.5.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Effect 3.5 (Release)
+excerpt: Release post highlighting new additions and changes
+date: 2024-07-10
+authors:
+  - name: Tim Smart
+    description: Founding Engineer
+    avatar: https://pbs.twimg.com/profile_images/3642301555/77cc6dab66dc94fdd77e503fd6d1c1cf_400x400.jpeg
+    twitter: https://twitter.com/tim_smart
+---
+
+Effect 3.5.0 has been released! This release includes a number of new features
+and improvements. Here's a summary of what's new:
+
+## Better support for Error.cause
+
+If you add a `cause` property to `Data.Error` or `Data.TaggedError`, it will now be properly
+forwarded to the `cause` property of the `Error` instance.
+
+```typescript
+import { Data } from "effect"
+
+class MyError extends Data.Error<{ cause: Error }> {}
+```
+
+If you `Effect.log` a `Cause` containing an error with a `cause` property, it will now be
+visible in the log output.
+
+## @effect/sql-d1
+
+The `@effect/sql-d1` package has been released. This package provides `@effect/sql` support
+for Cloudflare's D1 database.
```

Comment:
Let's add a shoutout to ecyrbe here.

Reply:
Fair. Although I think there's a difference between changelog & commit attribution (which we already do) and acknowleding significant contributions by singling out (especially new) contributors for such in the release posts. I'd argue that by attributing everything in the release post we would actually water down the significance of such a shoutout. I think it'd be a nice gesture.

Reply:
Thoughts @schickling @mikearnaldi ?

Reply:
Would be a first, I don't see the need imho

Reply:
Fine. Let's take the broader discussion around that elsewhere for now.